projects/core/tokens/icon-resolver.ts

```diff
@@ -23,12 +23,10 @@ export function tuiInjectIconResolver(): TuiStringHandler<string> {
     const icons = inject(TUI_ICON_REGISTRY);
     const resolver = inject(TUI_ICON_RESOLVER);
 
-    return (icon) => (!icon || icon.includes('/') ? icon : icons[icon] || resolver(icon));
+    return (icon) =>
+        !icon || icon.includes('/') ? icon : (icons[icon] ?? resolver(icon));
```

Comment:
For an inexperienced user, this looks rather cumbersome, I would generally rewrite it via if..

Reply:
I think it's ok. We typically write ternary every time we return value that can be expressed as `a ?  b : c`, even if there are some brackets involved.